### DISCUSSION TITLE
Does anyone know if there are plans to add electrical light sources (light poles or just light bulbs for example)?

### DISCUSSION THREAD

**Discussion by NEKO-001** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1140] [Q&A]
We are building factories with our friends and we are faced with the fact that we cannot beautifully illuminate the asphalt roads and workshops.

**Comment by JameH2** [comment]
Possible electrical sources in ntm:
RBMK
ZIRNOX
Watz powerplant
F-Watz (deprecated)
Steam turbine
Turbofan
Combustion generator
I-gen
Industrial combustion engine
Radial piston engine 
DFC
Flare stack
Stirling engine
Heavy stirling engine 
Radioisotope radiolysis machine
Radiation powered engine
RTG
ZPE
Geothermal generator
Diesel generator
Large nuclear reactor

I think there are plenty of electrical sources.

As for powered lights, i like the idea of a city going into blackout when something blows up, but its up to bob

**Reply by NEKO-001** [reply]
I meant only electric light sources, damn language barrier)
